packages/office-ui-fabric-react/src/components/List/examples/List.Basic.Example.tsx

```diff
@@ -72,53 +54,32 @@ const classNames: IListBasicExampleClassObject = mergeStyleSets({
     flexShrink: 0,
   },
 });
+const onRenderCell = (item: IExampleItem, index: number | undefined): JSX.Element => {
+  return (
+    <div className={classNames.itemCell} data-is-focusable={true}>
+      <Image className={classNames.itemImage} src={item.thumbnail} width={50} height={50} imageFit={ImageFit.cover} />
+      <div className={classNames.itemContent}>
+        <div className={classNames.itemName}>{item.name}</div>
+        <div className={classNames.itemIndex}>{`Item ${index}`}</div>
+        <div>{item.description}</div>
+      </div>
+      <Icon className={classNames.chevron} iconName={getRTL() ? 'ChevronLeft' : 'ChevronRight'} />
+    </div>
+  );
+};
 
-export class ListBasicExample extends React.Component<IListBasicExampleProps, IListBasicExampleState> {
-  private _originalItems: IExampleItem[];
-
-  constructor(props: IListBasicExampleProps) {
-    super(props);
-
-    this._originalItems = props.items || createListItems(5000);
-    this.state = {
-      filterText: '',
-      items: this._originalItems,
-    };
-  }
-
-  public render(): JSX.Element {
-    const { items = [] } = this.state;
-    const resultCountText =
-      items.length === this._originalItems.length ? '' : ` (${items.length} of ${this._originalItems.length} shown)`;
-
-    return (
-      <FocusZone direction={FocusZoneDirection.vertical}>
-        <TextField label={'Filter by name' + resultCountText} onChange={this._onFilterChanged} />
-        <List items={items} onRenderCell={this._onRenderCell} />
-      </FocusZone>
-    );
-  }
-
-  private _onFilterChanged = (_: any, text: string): void => {
-    this.setState({
-      filterText: text,
-      items: text
-        ? this._originalItems.filter(item => item.name.toLowerCase().indexOf(text.toLowerCase()) >= 0)
-        : this._originalItems,
-    });
+export const ListBasicExample: React.FunctionComponent = () => {
+  const [originalItems] = React.useState(() => createListItems(5000));
```

Comment:
You should do `useConst` here instead, passing in a callback (this is a perfect example of why I made it):
```suggestion
  const [originalItems] = useConst(() => createListItems(5000));
```

Same in all other list examples. 

You should also check to see how long the page takes to render now--as much as we generally want to avoid passing props to examples, not doing so in this case may cause a perf problem. If it is a perf problem, we could consider adding a memoized version of `createListItems` to `example-data`.